### PR TITLE
Add channel descriptions to the settings as they are in the wizard

### PR DESCRIPTION
**Pull request by irlaec**
(!) This branch was created from the branch f_whatsnew_betachannel, so no need to merge that one if we merge this.

Apart from adding the info about the channels, I made the "Check for update now" button sticky, so it is always visible:
Screenshot from 2019-05-10 12-09-09

### CODE REVIEW

**Comment by irlaec** [comment]
[Softare channel explanation](https://trello.com/c/ztDKOoCW/112-softare-channel-explanation)

File: octoprint_mrbeam/__init__.py [hunk 1]
```diff
@@ -365,7 +365,9 @@ def on_settings_save(self, data):
 			if "focusReminder" in data:
 				self._settings.set_boolean(["focusReminder"], data["focusReminder"])
 			if "dev" in data and "software_tier" in data['dev']:
+				self._logger.info("ANDYTEST on_settings_save software_tier: START")
```
Comment (andywerner) [inline octoprint_mrbeam/__init__.py]: remove this

File: octoprint_mrbeam/__init__.py [hunk 2]
```diff
@@ -365,7 +365,9 @@ def on_settings_save(self, data):
...
 				switch_software_channel(self, data["dev"]["software_tier"])
+				self._logger.info("ANDYTEST on_settings_save software_tier: DONE")
```
Comment (andywerner) [inline octoprint_mrbeam/__init__.py]: remove this

**Comment by irlaec** [comment]
New look:
image

**Review by andywerner — APPROVED** [review]

**Pull request merged**